src/host/node_connections.h

```diff
@@ -143,10 +145,33 @@ namespace asynchost
 
       void reconnect()
       {
-        auto s = parent.find(node);
+        // Use uv_timer_t to reconnect after a short delay
```

Comment:
Previously, if we tried to talk to another node and got a failure, this got very spammy - we looped back into `reconnect` directly and retried immediately, indefinitely. The most common case for this failure is that the other node really is dead (eg - at the end of a test), so retrying is useless. Now we at least have a delay between retries - shouldn't cause a noticeable slowdown if the retry succeeds, and makes the logs useable even with a persistent failure.